src/ui/settings/ViewLanguage.qml

```diff
@@ -120,7 +125,8 @@ Item {
                     objectName: "settingsSystemLanguageToggle"
                     toolTipTitle: {
                         if (checked) {
-                           //% "Disable to select different language"
+                           //% "Disable to select a different language"
```

Comment:
Not sure if it's just me, but "Disable to select different language" sounds a bit off.